doc/hopauth_guide.md

```diff
@@ -74,112 +77,130 @@ Logging in goes through the CILogon service and your institutional single sign o
+[![The management page for a topic](hopauth_screenshots/topic_management_g.png)](hopauth_screenshots/topic_management_g.png)
 
-A topic is always accessible to all members of the group which owns it, but by default it is not accessible to any other users. 
+A topic is always accessible to all members of the group which owns it, but by default it is not accessible to any other users.
 
 ### Granting Other Groups Access to Topics
 
-Since, by default, Hopskotch user who are not members of a group cannot access that group's topics, it may be desirable to grant such access in order to share data. 
+Since, by default, Hopskotch user who are not members of a group cannot access that group's topics, it may be desirable to grant such access in order to share data.
 
-To manage access to a topic, locate the topic in the 'Topics' section of the group page, and click the 'Edit' button. This will open the topic's management page. 
+To manage access to a topic, locate the topic in the 'Manage Group Topics' page, and click the 'Edit' button. This will open the topic's management page. In the 'Manage Group Topics' page you can also delete the topic entirely.
 
 [![The management page for a topic](hopauth_screenshots/topic_management.png)](hopauth_screenshots/topic_management.png)
 
-This page shows all currently granted permissions (at the group level) for the topic, gives the option to add additional permissions, and to delete the topic entirely. The tale of permissions will always show the owning group having 'All' permission for the topic; this permission cannot be changed or removed as long as the topic exists. 
+The 'Manage Topic' page shows all currently granted permissions (read/write) for the topic (at the group level), gives the option to modify those permissions. The owning group will have 'All' permission for the topic; this permission cannot be changed or removed as long as the topic exists. In this page you can also modify the topic's description, define its visibility and set the archiving option.
 
-The first option is to grant specific levels of access to specific groups. This can be done by choosing the permission type and group to grant access from the lists in the 'Add Permission' section, and then pressing the 'Add permission' button. One this is done, members of the chosen group will have the option to add this permission to their credentials. The permission will also be listed in the 'Active Group Permissions' section of the topics management page:
+One option to grant specific levels of access to specific groups is adding a group to a topic. This can be done by clicking the 'Add' button next to a group's name in the 'Available Groups' section, then you will have to set the read/write permissions, once the group appears in the 'Added Groups' section. One this is done, members of the chosen group will have the option to add this permission to their credentials. :
```

Comment:
'adding a group to a topic' should be 'granting a group access to a topic'. The interface itself may need some of this language fixed at some point.